modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/DocumentResourceTest.java

```diff
@@ -238,11 +252,1003 @@ protected void assertValid(
 			_read("http://localhost:8080" + document.getContentUrl()));
 	}
 
+	@Override
+	protected boolean equals(Document document1, Document document2) {
+		if (document1 == document2) {
```

Comment:
Idea:

```
	@Override
	protected boolean equals(Document document1, Document document2) {
if (!FeatureFlagManagerUtil.isEnabled(
					testCompany.getCompanyId(), "LPD-10701") ){
return super.equals();
}
```

and then this code without the FFs


This way we can be sure on the removal that this code should be the reemplace once the removal of the FF